Examples/Movies/android/app/src/main/java/com/facebook/react/movies/MoviesActivity.java

```diff
@@ -35,7 +36,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
     mReactInstanceManager = ReactInstanceManager.builder()
         .setApplication(getApplication())
-        .setBundleAssetName("MoviesApp.android.bundle")
+        .setJSBundleLoader(JSBundleLoader.createAssetLoader("MoviesApp.android.bundle"))
```

Comment:
Sorry for not being super clear. One use case we're looking at is changing the `bundleFile` at run time, and then "reloading" the React instance to load from the new bundle location. (this is possible on the iOS side via the delegate methods)

Do you think the "setBundleFile" API could support this? Should we expose a method for changing the `bundleFile` after a ReactInstance has been instantiated?